app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,28 @@
+class ExpensesController < ApplicationController
+  def index
+    @expense = Expense.all
```

Comment:
We shouldn't initialize redundant actions. There is no place in your app where you need `expanses#index` now :).